site/tests/app/controllers/admin/PlagiarismControllerTester.php

```diff
@@ -0,0 +1,34 @@
+<?php namespace tests\app\controllers\admin;
+
+use app\libraries\Core;
+use tests\BaseUnitTest;
+use app\controllers\admin\PlagiarismController;
+
+class PlagiarismControllerTester extends BaseUnitTest
+{
+   /** @var PlagiarismController */
+    protected $controller;
+
+    /** @var Core */
+    protected $core;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+
+        $this->core = $this->createMockCore([], [], [
+            'getGradeableConfig'  => [],
```

Comment:
You can't return an empty array as it expects an object with a getTitle function. You need to create a mock object that returns a string? I think? on call to it.